doc/whats_new/v1.4.rst

```diff
@@ -20,6 +20,19 @@ Version 1.4.1
 
 **In Development**
 
+Metadata Routing
+----------------
+
+- |FIX| Fix routing issue with :class:`~compose.ColumnTransformer` when used
+  inside another meta-estimator.
+  :pr:`28188` by `Adrin Jalali`_.
+
+DataFrame Support
+-----------------
+
+- |Enhancement| Pandas and Polars dataframe are validated directly without ducktyping
```

Comment:
Is this part of this PR?

Reply:
yes, was under the wrong section, but I had forgotten to remove the duplicate. Fixed now.